tensorflow/contrib/data/python/ops/interleave_ops.py

```diff
@@ -200,10 +200,10 @@ def sample_from_datasets(datasets, weights=None, seed=None):
 
   Args:
     datasets: A list of @{tf.data.Dataset} objects with compatible structure.
-    weights: (Optional.) A list of `len(datasets)` floating-point values,
-      where `weights[i]` represents the probability with which an element
-      should be sampled from `datasets[i]`. Defaults to a uniform distribution
-      across `datasets`.
+    weights: (Optional.) A list of `len(datasets)` floating-point values or a
+      @{tf.data.Dataset} object, where `weights[i]` represents the probability
+      with which an element should be sampled from `datasets[i]`. Defaults to a
+      uniform distribution across `datasets`.
```

Comment:
IIUC, the interpretation of a `Dataset` argument is a bit different from what's written here. Is it the case that each element of (the `Dataset` version of) `weights` is a vector of `len(datasets)` and the ith element determines the weights for sampling the ith output element?

(If so, can you please update the comment to clarify?)

Reply:
@mrry Yes you are right. I updated the docstring to (hopefully) clarify.